### ISSUE TITLE
Virtualbox (NP) (7.0.2) - Can't shim 'VBoxSDL.exe': File doesn't exist.

### ISSUE THREAD

**Issue by giflw** [ScoopInstaller/Nonportable#92]
## Bug Report

**Package Name:** virtualbox-np 7.0.2 and virtualbox-with-extension-pack-np 7.0.2

### Current Behaviour

Not installing because of missing VBoxSDL executable (same as #72 and #73).

### Expected Behaviour

Package be installed.

### Additional context/output

```
Creating shim for 'VBoxSDL'.
Can't shim 'VBoxSDL.exe': File doesn't exist.
```

### Possible Solution

Remove shim for VBoxSDL.

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:**

Major  Minor  Patch  PreReleaseLabel BuildLabel
-----  -----  -----  --------------- ----------
7      2      7
and
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      19041  1682

**Additional software:** Windows Terminal

#### Scoop Configuration

```json
{
  "last_update": "2022-11-07T21:35:46.2656123-03:00",
  "scoop_branch": "master",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop"
}
```

**Comment by GalaxySnail** [comment]
After #73 merged, it has been fixed in this repo. But I found that my `~/scoop/buckets/nonportable` is pointing to another repo: [TheRandomLabs/scoop-nonportable](https://github.com/TheRandomLabs/scoop-nonportable), which is still affected by this issue. Using this repo instead of that repo can solve it.

**Comment by giflw** [comment]
Same error on v7.0.4

```
➜ scoop install virtualbox-np
Installing 'virtualbox-np' (7.0.4) [64bit] from nonportable bucket
VirtualBox-7.0.4-154605-Win.exe (105,2 MB) [==================================================================] 100%
Checking hash of VirtualBox-7.0.4-154605-Win.exe ... ok.
Running installer script...
Linking ~\scoop\apps\virtualbox-np\current => ~\scoop\apps\virtualbox-np\7.0.4
Creating shim for 'VBoxBalloonCtrl'.
Creating shim for 'VBoxBugReport'.
Creating shim for 'VBoxDTrace'.
Creating shim for 'VBoxExtPackHelperApp'.
Creating shim for 'VBoxHeadless'.
Creating shim for 'vbox-img'.
Creating shim for 'VBoxManage'.
Creating shim for 'VBoxNetDHCP'.
Creating shim for 'VBoxNetNAT'.
Creating shim for 'VBoxSDL'.
Can't shim 'VBoxSDL.exe': File doesn't exist.
```

**Comment by GalaxySnail** [comment]
It can be fixed by running this:

```
scoop bucket rm nonportable
scoop bucket add nonportable
```

- https://github.com/TheRandomLabs/scoop-nonportable/issues/361

**Comment by giflw** [comment]
Still cannot install it

```
➜ scoop bucket list

Name        Source                                             Updated             Manifests
----        ------                                             -------             ---------
extras      https://github.com/ScoopInstaller/Extras           19/11/2022 05:31:59      1731
java        https://github.com/ScoopInstaller/Java             19/11/2022 05:21:51       253
main        https://github.com/ScoopInstaller/Main             19/11/2022 09:30:33      1118
nerd-fonts  https://github.com/matthewjberger/scoop-nerd-fonts 19/11/2022 08:28:10       208
nonportable https://github.com/ScoopInstaller/Nonportable      19/11/2022 01:42:37       104


➜ sudo scoop install virtualbox-np
Installing 'virtualbox-np' (7.0.4) [64bit] from nonportable bucket
VirtualBox-7.0.4-154605-Win.exe (105,2 MB) [================================================================================================================] 100%
Checking hash of VirtualBox-7.0.4-154605-Win.exe ... ok.
Running installer script...
Remove-Item:
Line |
   7 |  Remove-Item "$env:PUBLIC\Desktop\Oracle VM VirtualBox.lnk"
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\Users\Public\Desktop\Oracle VM VirtualBox.lnk' because it does not exist.
Remove-Item:
Line |
   8 |  Remove-Item -Recurse "$([Environment]::GetFolderPath('commonstartmenu …
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Oracle VM VirtualBox' because it does not exist.
Linking ~\scoop\apps\virtualbox-np\current => ~\scoop\apps\virtualbox-np\7.0.4
Creating shim for 'VBoxBalloonCtrl'.
Creating shim for 'VBoxBugReport'.
Creating shim for 'VBoxDTrace'.
Creating shim for 'VBoxExtPackHelperApp'.
Creating shim for 'VBoxHeadless'.
Creating shim for 'vbox-img'.
Creating shim for 'VBoxManage'.
Creating shim for 'VBoxNetDHCP'.
Creating shim for 'VBoxNetNAT'.
Creating shim for 'VBoxSDS'.
Can't shim 'VBoxSDS.exe': File doesn't exist.
```

**Comment by jcwillox** [comment]
Not sure why I opened another issue https://github.com/ScoopInstaller/Nonportable/issues/145, I closed it as it seems to be the same issue.  I could've just commented my findings here 🤦‍♂️, its too hot here to think straight 😄.

Anyways I looked into the manifest/install and from what I can tell it seems like the installer and uninstaller scripts are broken. It doesn't actually install VirtualBox (well extract the setup.exe), which is why the shims are failing, none of them exist.
